Sources/RealDeviceMapLib/Modell/SpawnPoint.swift

```diff
@@ -233,6 +302,8 @@ public class SpawnPoint: JSONConvertibleObject {
             let updated = result[3] as! UInt32
             let lastSeen = result[4] as! UInt32
             let despawnSecond = result[5] as? UInt16
+            let spawnInfo = result[6] as! UInt32
+            let firstSeenTimestamp = result[7] as! UInt32
```

Comment:
@madBeavis unfortunately you are adding this column in DB and set it to `null` for all, but here you load them as `non-nullable` and this breaks the workflow. must this be initialized as `null` on migration or can we use `updated` timestamp instead?

Reply:
good catch, i made the changes

thanks for patience and help